operations/helm/charts/mimir-distributed/values.yaml

```
@@ -653,6 +721,56 @@ ingester:
   env: []
   extraEnvFrom: []
 
+  # -- Options to configure zone aware replication for ingester
+  zoneAwareReplication:
+    # -- Enable zone aware replication for ingester
+    enabled: true
+    # -- Maximum number of ingesters that can be unavailable per zone during rollout
+    maxUnavailable: 25
+    # -- topologyKey to use in pod anti-affinity. If unset, no anti-affinity rules are generated. If set, the generated anti-affinity rule makes sure that pods from different zones do not mix.
```

nit, can reject if you think this is too much explanaiton
```suggestion
    # -- topologyKey to use in pod anti-affinity. If unset, no anti-affinity rules are generated. If set, the generated anti-affinity rule makes sure that pods from different zones do not mix on nodes with the same topology key value.
```